src/assets/styles/_base.scss

```diff
@@ -18,6 +18,7 @@
   align-items: center;
   justify-content: space-between;
   flex-basis: var(--calcite-app-header-min-height);
+  min-height: var(--calcite-app-header-min-height);
```

Comment:
is this and flex-basis both needed? Seems like one or the other would be redundant. Or are we using the same CSS variable for two different purposes? If it's the latter, can we rename the variable to prevent confusion?